git/working.go

```diff
@@ -89,6 +90,12 @@ func (r *Repo) Clone(ctx context.Context, conf Config) (*Checkout, error) {
 	}
 	r.mu.RUnlock()
 
+	if conf.GitSecret {
```

Comment:
I think Flux should log an error at startup if the `git-secret` is enabled but `git-gpg-key-import` is empty. I would also add to the `git-secret` flag description that importing the GPG key is required.

Reply:
I think it's possible, though a major pain, to mount a GPG keyring rather than importing keys. So _not_ using the import does not necessarily mean there are no keys for `git secret` to use.

On the other hand: I anticipate mistakes here, so I do agree with Stefan's advice -- downgrade it to a warning, perhaps.

Reply:
Hello, thanks for your comments, I tried to implement the warn message and completed the description, is it correct like this?

Reply:
Yes thanks